routes/usersBooks.js

```diff
@@ -0,0 +1,7 @@
+var express = require('express');
+var router = express.Router();
+var addUserBook = require('../controllers/userBooks/new')
+
+router.get('/:id', addUserBook.add);
```

Comment:
Ah is this where  .then((book) => res.redirect(`/usersBooks/${book.id}`)) comes in? I think this is a messy flow...